docs/docs/detectors/18-unrestricted-transfer-from.md

```diff
@@ -1,60 +1,77 @@
-# Unrestricted Transfer From
+# Unrestricted transfer from
 
-### What it does
+## Description
 
-It warns you if a `transfer_from` function is called with a user-defined parameter in the `from` field.
+- Category: `Validations and error handling`
+- Severity: `High`
+- Detectors: [`unrestricted-transfer-from`](https://github.com/CoinFabrik/scout-soroban/tree/main/detectors/unrestricted-transfer-from)
+- Test Cases: [`unrestricted-transfer-from-1`](https://github.com/CoinFabrik/scout-soroban/tree/main/test-cases/unrestricted-transfer-from/unrestricted-transfer-from-1)
 
-### Why is this bad?
+Allowing unrestricted `transfer_from` operations poses a significant issue. When `from` arguments for that function is provided directly by the user, this might enable the withdrawal of funds from any actor with token approval on the contract. 
 
-An user Alice can approve a contract to spend their tokens. An user Bob can call that contract, use that allowance to send themselves Alice's tokens. 
+## Why is this bad?
 
+An user Alice can approve a contract to spend their tokens. An user Bob can call that contract, use that allowance to send themselves Alice's tokens.
```

Comment:
Creo que esta explicación podría estar más clara. Podríamos poner algo así: "The absence of proper authorization checks for sensitive operations, like `transfer_form`, can lead to the loss of funds or other undesired consequences. For example, if a user, Alice, approves a contract to spend her tokens, and the contract lacks proper authorization checks, another user, Bob, could invoke the contract and potentially transfer Alice's tokens to himself without her explicit consent.